sui/deploy-contract.js

```diff
@@ -269,16 +271,17 @@ async function upgrade(keypair, client, supportedPackage, policy, config, chain,
     const contractConfig = contractsConfig?.[packageName];
 
     validateParameters({ isNonEmptyString: { packageName } });
+    const moveDir = `${__dirname}/move`;
 
     if (packageDependencies) {
         for (const dependencies of packageDependencies) {
```

Comment:
can you look into doing this dynamically? read the Move.toml and get all the local dependencies, then read from config. separate PR makes sense for it

Reply:
add jira ticket here https://axelarnetwork.atlassian.net/browse/AXE-4977